edits.go

```diff
@@ -30,7 +30,10 @@ type EditsResponse struct {
 	Choices []EditsChoice `json:"choices"`
 }
 
-// Perform an API call to the Edits endpoint.
+// Edits Perform an API call to the Edits endpoint.
+/* Deprecated: Users of the Edits API and its associated models (e.g., text-davinci-edit-001 or code-davinci-edit-001)
+will need to migrate to GPT-3.5 Turbo by January 4, 2024.
```

Comment:
@liushuangls I thought a comment should note that CreateChatCompletion func (Chat Completions API) is an alternative.

> Users of the Edits API and its associated models (e.g., text-davinci-edit-001 or code-davinci-edit-001) will need to migrate to GPT-3.5 Turbo by January 4, 2024. The Edits API beta was an early exploratory API, meant to enable developers to return an edited version of the prompt based on instructions. We took the feedback from the Edits API into account when developing gpt-3.5-turbo and the Chat Completions API, which can now be used for the same purpose:

Refs: https://openai.com/blog/gpt-4-api-general-availability

Reply:
@vvatanabe I added it, thanks for your suggestion.

Reply:
@liushuangls You are always quick to work and helpful!